src/index.js

```diff
@@ -1,15 +1,11 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
```

Comment:
ReactDOM is needed for line 36 to work otherwise the app won't load and you will see an error in the devconsole saying `ReactDOM is not defined`